src/Altinn.AccessManagement.Core/Services/DelegationsService.cs

```diff
@@ -224,6 +224,86 @@ public async Task<List<Delegation>> GetMaskinportenSchemaDelegations(string supp
             return await GetAllMaskinportenSchemaDelegations(supplierPartyId, consumerPartyId, scope);
         }
 
+        /// <inheritdoc/>
+        public async Task<DelegationOutput> RevokeMaskinportenDelegation(int authenticatedUserId, DelegationInput delegation)
```

Comment:
Wondering if we should rename the models used here for both delegation and revoke:
DelegationInput -> DelegationLookup ?
DelegationOutput -> DelegationResult/DelegationActionResult ?

Reply:
Models has been renamed DelegationLookup and DelegationActionResult